src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets

```diff
@@ -1,26 +1,10 @@
 <Project>
-
   <PropertyGroup>
     <PublishWithAspNetCoreTargetManifest Condition="'$(PublishWithAspNetCoreTargetManifest)'==''">true</PublishWithAspNetCoreTargetManifest>
+    <ManifestRuntimeIdentifier>$(AspNetCoreTargetManifestRuntimeIdentifier)</ManifestRuntimeIdentifier>
```

Comment:
`<ManifestRuntimeIdentifier Condition="'$(PublishWithAspNetCoreTargetManifest)'=='true' and '$(PublishableProject)'=='true'">$(AspNetCoreTargetManifestRuntimeIdentifier)</ManifestRuntimeIdentifier>`